### PR TITLE
Edit Functionality Working - fixed

### PR DESCRIPTION
**Pull request by suvarnakale**

### CODE REVIEW

**Review by Pratikshakhandagale — APPROVED** [review]

File: src/app/issure/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -80,12 +80,13 @@ <h6 class="modal-title primary-text" id="prewiewProfile">Edit issuer details</h6
...
-                        <form [formGroup]="form2" (ngSubmit)="submit()">
+                        <form [formGroup]="form2" (ngSubmit)="submit()" id="myForm">
                             <formly-form [model]="model" [fields]="fields" [options]="options" [form]="form2">
                             </formly-form>
+                            <!-- <app-forms form={{form}} identifier={{identifier}} modal="true"></app-forms> -->
                             <button type="button" data-dismiss="modal" aria-label="Close"
                                 class="btn btn-sec-bg">Cancel</button>
-                            <button type="submit" class="btn btn-primary float-end submit-button"
+                            <button type="submit" class="btn btn-primary float-end submit-button" id="dismissButton"
                                 [disabled]="!form2.valid">Save </button>
```
Comment (Pratikshakhandagale) [inline src/app/issure/dashboard/dashboard.component.html:90]: @suvarnakale you are not doing anything using this id , remove it or set the proper button id name.

File: src/app/issure/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -142,17 +151,32 @@ export class DashboardComponent implements OnInit {
...
+      modalBackdrop.remove();
+    }
+  }
+
 
   submit() {
-    this.generalService.putData('/issuer', this.model['osid'], this.model).subscribe((res) => {
+    this.generalService.putData('/Issuer', this.model['osid'], this.model).subscribe((res) => {
+      this.temp = res;
+      console.log(this.temp);
```
Comment (Pratikshakhandagale) [inline src/app/issure/dashboard/dashboard.component.ts]: @suvarnakale  Remove console from code

File: src/app/issure/dashboard/dashboard.component.html (hunk 1 above)
Comment (suvarnakale) [reply src/app/issure/dashboard/dashboard.component.html:90]: Okay

File: src/app/issure/dashboard/dashboard.component.ts (hunk 2 above)
Comment (suvarnakale) [reply src/app/issure/dashboard/dashboard.component.ts]: okay

**Pull request merged**